compiler/optimizer/SinkStores.cpp

```diff
@@ -619,19 +613,6 @@ int32_t TR_SinkStores::performStoreSinking()
    _placementsForEdgesToBlock = (TR_EdgeStorePlacementList **) trMemory()->allocateStackMemory(arraySize);
    memset(_placementsForEdgesToBlock, 0, arraySize);
 
-   // some extra meta data is needed if stores with indirect loads are going to be sunk
-   if (sinkStoresWithIndirectLoads())
```

Comment:
Note to reviewers:  this function always returned false, hence the code it guards was removed.